Bio/SeqUtils/MeltingTemp.py

```diff
@@ -654,7 +661,8 @@ def Tm_Wallace(seq, check=True, strict=True):
     practical approach, 33-50) is often used as rule of thumb for approximate
     Tm calculations for primers of 14 to 20 nt length.
 
-    Non-DNA characters (e.g., E, F, J, !, 1, etc) are ignored by this method.
+    Non-alphabetic characters (numbers, spacers, symbols, etc) are ignored.
+    Non-DNA alphabetic characters (e.g., E, F, J, etc) will raise an error if check=True.
```

Comment:
Why would anyone want `check=False`?

Reply:
Having check=False makes it run nearly twice as fast. 2.8s vs. 4.9s for ~1 million calls (using the examples in the tests)

Reply:
You could count all acceptable letters, and see if the sum equals the length of the string.
Btw, bytes is faster than string for this calculation.

Reply:
Melting temperatures can be calculated for non-standard bases. In these cases you want to use additional characters to represent them and thus would turn `check` off. There is an example in the docstring, if I'm not mistaken.